test/jdk/com/sun/security/util/math/intpoly/MontgomeryPolynomialFuzzTest.java

```diff
@@ -23,9 +23,8 @@
  */
 
 import java.util.Random;
-import sun.security.util.math.IntegerMontgomeryFieldModuloP;
-import sun.security.util.math.ImmutableIntegerModuloP;
 import java.math.BigInteger;
+import sun.security.util.math.*;
 import sun.security.util.math.intpoly.*;
 
 /*
```

Comment:
It is strange that there are two copies of the `@test` block.  Can you please remove one of them, unless you are seeing a difference that I do not

Reply:
Turns out I do need both `@test`; (otherwise `make test TEST=...MontgomeryPolynomialFuzzTest.java` runs fewer tests). Seems other tests do the same.

I did add a (better?) comment to the summary tag.